Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM php:7.0-apache
+
+LABEL maintainer="devops@onix-systems.com"
+
+ENV COMPOSER_ALLOW_SUPERUSER=1 \
+    MYSQL_HOSTNAME=localhost \
+    MYSQL_USER=user \
+    MYSQL_PASSWORD=123456 \
+    MYSQL_DATABASE=rename \
+    CONFIG_FILE=/var/www/html/config/config.php
+
```

Comment:
Why you use the hardcoded variables? I think you should use the one place to store credentials. I seen below "credentials" file, maybe better use it?

Reply:
These are default values for docker image